index.html

```diff
@@ -2812,43 +2812,33 @@ <h3><dfn>Writing or pushing content</dfn></h3>
       </p>
       <ol class=algorithm id="validate-external-type">
         <li>
-          If |input:string| is not a {{USVString}} or it is empty, or its
+          If |input:string| is not an [=ASCII string=] or it is empty, or its
```

Comment:
Nit: "If |input:string| is not an [=ASCII string=], is empty, or its length exceeds 255 bytes, return `false`."

Reply:
Fixed, thanks.